tox.ini

```diff
@@ -23,3 +23,17 @@ commands =
     flake8 .
     bandit -r .
     mypy pydax
+
+[pytest]
+addopts = --doctest-modules
+
+[coverage:run]
+source = pydax
+
+[coverage:report]
+fail_under = 100
+show_missing = True
+
+[flake8]
+max-line-length = 120
+exclude = .eggs,.git,__pycache__,.tox
```

Comment:
How are `coverage`, `flake8`, and `pytest` capable of finding these configurations in `tox.ini` even when we run their commands outside of a tox environment?

Reply:
Because they also see `tox.ini` as their configuration files, but not the others :smile_cat: